### PR TITLE
added unread colour, fixed sizes to default

### PR DESCRIPTION
**Pull request by ekohilas**

### CODE REVIEW

File: messenger-hover.css [hunk 1]
```diff
@@ -33,11 +38,12 @@
     -webkit-transition: -webkit-flex-basis 0.1s, min-width 0.1s;
 }
 ._1enh:hover {
-    flex-basis: 20%;
-    min-width: 320px;
+    flex-basis: 25%;
+    max-width: 420px;
```
Comment (auscompgeek) [inline messenger-hover.css:42]: Why was this line added back in?

File: messenger-hover.css [hunk 2]
```diff
@@ -11,6 +11,11 @@
     content: ''
 }
 
+/*unread color*/
+._1ht3._1ht1 {
+    background-color: rgba(0, 132, 255, .2);
+    -webkit-transition: background-color .5s
```
Comment (auscompgeek) [inline messenger-hover.css:17]: ಠ_ಠ

File: messenger-hover.css [hunk 3]
```diff
@@ -33,11 +38,12 @@
     -webkit-transition: -webkit-flex-basis 0.1s, min-width 0.1s;
 }
 ._1enh:hover {
-    flex-basis: 20%;
-    min-width: 320px;
+    flex-basis: 25%;
+    max-width: 420px;
+	min-width: 320px;
```
Comment (auscompgeek) [inline messenger-hover.css:43]: wtf

**Comment by auscompgeek** [comment]
why on earth does this pull request exist anyway

**Comment by ekohilas** [comment]
No idea
